### PR TITLE
Add indent option to JSONResponse

### PR DESCRIPTION
**Pull request by Nebula1213**
New JSONResponse class:

https://pastecord.com/apemokopur.py

### CODE REVIEW

File: starlette/responses.py [hunk 1]
```diff
@@ -153,12 +153,23 @@ class PlainTextResponse(Response):
...
     media_type = "application/json"
 
+    def __init__(
+        self, 
+        content: typing.Any = None, 
+        status_code: int = 200, 
+        headers: dict = None, 
+        media_type: str = None, 
+        background: BackgroundTask = None, 
+        indent: int = None) -> None:
```
Comment (aminalaee) [inline starlette/responses.py]: I think as seen [here](https://github.com/encode/starlette/runs/3392828918) `black` will format this differently.
You can run `scripts/lint` locally.

Comment (Nebula1213) [reply starlette/responses.py]: the newest commit should fix the formatting issue

**Comment by Kludex** [comment]
What's the reasoning behind this? I don't see a good reason for this, could you enlighten me? :thinking:

**Comment by Kludex** [comment]
As the author didn't reply in a long time, and there's no written reasoning behind this implementation, I'm closing this PR.

**Pull request closed without merging**